packages/network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [18.1.1]
+
+### Changed
```

Comment:
Nit: Maybe better described as "Added"? These are new exports, no existing functionality has changed with this commit.